dockerfiles/hpcc/values.yaml

```diff
@@ -63,7 +63,7 @@ eclagent:
   ## replicas indicates how many eclagent pods should be started
   replicas: 1
   ## maxActive controls how many workunits may be active at once (per replica)
-  maxActive: 100
```

Comment:
Also change default in agentexec.cpp?

Reply:
I was in two minds about this.

There's a difference between what we want the default to be if you don't specify in your helm chart, and what you want the default to be in values.yaml (representing a minimal system expected to be run on dodker desktop or similar).

If you are running on a full cloud and want to scale up according to load, 100 may be fine.

There's also an argument that we should be setting a similar limit for the child process case (in agentexec) as the k8s job case. Does setting childProcess imply you expect less parallelism, or that you expect smaller jobs?